options/options.js

```diff
@@ -16,22 +23,27 @@ function save_options() {
 
 // Restores option state using the preferences stored in chrome.storage.
 function restore_options() {
-    chrome.storage.local.get('MB', function(items) {
-        var MB;
-        if(items.MB !== undefined) {
-            MB = items.MB;
+    chrome.storage.local.get(['DecoderMemoryLimitInMegabytes', 'DebugOutput'], function(items) {
+    console.log(items.DecoderMemoryLimitInMegabytes);
+        if(items.DecoderMemoryLimitInMegabytes !== undefined) {
+            document.getElementById('DecoderMemoryLimitInMegabytes').value = items.DecoderMemoryLimitInMegabytes;
+        } else {
+            document.getElementById('DecoderMemoryLimitInMegabytes').value = DecoderMemoryLimitInMegabytes;
+        }
+        
+        if(items.DebugOutput !== undefined) {
+            document.getElementById('DebugOutput').checked = items.DebugOutput;
         } else {
-            MB = "32";
+            document.getElementById('DebugOutput').checked = DebugOutput;
```

Comment:
Here is a good example on the naming issue. It is not clear what the `DebugOutput` value you are using is coming from right now, looking at this line alone. Using a name like `DebugOutputEnabledDefault` will indicate clearly that it is a default value instead.